### DISCUSSION TITLE
Editor skips objects when drag selecting too fast

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#16398] [Q&A] [answered]
As the title says.
Drag selecting slowly is fine.

https://user-images.githubusercontent.com/64581009/148749267-427cd357-1d7e-43ca-91e8-b71b2b01a2a8.mp4

~~Could be related to #16356 as well.~~

**Comment by bdach** [comment]
it's not related to the aforementioned issue. at any given point only a selected number of editor blueprints is visible to improve performance, but that means that if they aren't there then they won't get selected.

range selection probably needs to use editor beatmap for determining what to select rather than the blueprints.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16410
